src/main/java/de/azapps/kafkabackup/common/offset/OffsetSinkScheduler.java

```diff
@@ -0,0 +1,37 @@
+package de.azapps.kafkabackup.common.offset;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+public class OffsetSinkScheduler {
+    private static final Logger log = LoggerFactory.getLogger(OffsetSinkScheduler.class);
```

Comment:
@Sl4j ? :D

Reply:
Ah, ofc. I actually wrote it before the other review, and forgot to update this one 🤦‍♂️